src/main.rs

```diff
@@ -204,6 +212,13 @@ fn main() {
         _ => panic!("unsupported output format"),
     };
 
+    let init_method = match matches.value_of("init-method").unwrap() {
+        "random" => InitMethod::Random, 
```

Comment:
I might have already asked that, but does it make sense to have option to select smallest/largest eigenvectors?

With laplacian only one case make sense, right?

Reply:
Yep, I answered it [here](https://github.com/Synerise/cleora/pull/68#discussion_r957467257). In more detail though, the largest eigenvalues capture the "non-edges" (the edges in the complement graph) but feeding this through, let's say, a logistic regression model for example can still use this information. So in itself probably it is not the most useful, in the right circumstances it can be a powerful tool.